src/reporter/query_1T1ENA.py

```diff
@@ -57,25 +57,29 @@ def query_1T1ENA(entity_id,   # In Path
         return msg, 500
 
     if entities:
+        if len(entities) > 1:
+            import warnings
+            warnings.warn("Not expecting more than one result for a 1T1ENA.")
+
         if aggr_method:
-            index = []
+            if aggr_period:
```

Comment:
see above

Reply:
it's the same for all endpoints.

https://github.com/smartsdk/ngsi-timeseries-api/blob/bc8d0bcb97b3e3d2709649cc34d7add9240a1d4c/src/reporter/reporter.py#L286